README.md

```diff
@@ -25,90 +20,55 @@ Here are the method signatures as definied in the `StringPreference` class.
 
 ```java
 public class StringPreference {
-    public String get() { ... }
-    public boolean isSet() { ... }
-    public void set(String value) { ... }
-    public void delete() { ... }
+    public String get() {  }
+    public boolean isSet() {  }
+    public void set(String value) {  }
+    public void delete() {  }
 }
 ```
 
-Wrappers for frequently used classes can be found and added in the [wiki][typedpreferences-wiki]. Take a look!
+## Source
 
+Inspired by https://github.com/johnjohndoe/TypedPreferences - version 1.0
 
-## How to get the thing running?
-
-Add the library as a dependency to your `build.gradle` in case you use Gradle:
 
+## Dependency
+1. For using apprater module in sample app, include the source code and add the below dependencies in entry/build.gradle to generate hap/support.har.
 ```groovy
-dependencies {
-    compile "info.metadude.android:typed-preferences:{$version}"
+    dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar', '*.har'])
+    implementation project(path: ':library')
```

Comment:
Current lib name will cause an issue in Maven release. Refactor the lib module name from "Library" to "typedpreferences" in the project. Do the necessary changes throughout the project.